sympy/stats/crv.py

```diff
@@ -153,11 +153,54 @@ def __new__(cls, dist, size):
     @classmethod
     def _sample_scipy(cls, dist, size):
         """Sample from SciPy."""
-        # scipy does not require map as it can handle using custom distributions
-        from scipy.stats import rv_continuous
+        import scipy.stats
+        # scipy does not require map as it can handle using custom distributions.
+        # However, we will still use a map where we can.
+
+        # TODO: do this for drv.py and frv.py if necessary.
+        # TODO: add more distributions here if there are more
```

Comment:
Can you please do it for `drv.py` and `frv.py` in two different PRs?

Reply:
I'll only be able to get to it at the start of December. 

If that is too late (and the PRs need to go in the 1.7 release), then I'd suggest someone else take over for these.

Reply:
Shall I do for one ?

Reply:
Well the whole sampling architecture is refactored at #20494 
I would suggest to do after it's merged.